src/_tsd.d.ts

```diff
@@ -1,3 +1,3 @@
 /// <reference path="../typings/knockout/knockout.d.ts" />
-/// <reference path="../node_modules/definition-header/dist/index.d.ts" />
+/// <reference path="../node_modules/tsd/node_modules/definition-header/dist/index.d.ts" />
```

Comment:
> This change is part of #1 pull request, right?

Yes, I made the mistake to push with an old  commit.
I have to undo this commit and re-push right?